.github/workflows/macOS-12.yml

```diff
@@ -25,6 +26,22 @@ env:
   DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Resolve dependencies
+      run: swift package resolve
+    - name: GitHub Action for SwiftLint
+      uses: norio-nomura/action-swiftlint@3.2.1
+    - name: GitHub Action for SwiftLint with --strict
+      uses: norio-nomura/action-swiftlint@3.2.1
+      with:
+          args: --strict
+    - name: GitHub Action for SwiftLint (Only files changed in the PR)
```

Comment:
This will be the action we use in the future (only files changed in PR). 

The others were just added for testing.

Reply:
Maybe it worth to not extract this one into separate job? I mean, why isn't add this one as just a first step in SPM flow?

Reply:
Yea this was my first way of testing it but this action just runs on linux (ubuntu).

Reply:
Probably we can run it sequentially too. Then the other job won't even run if there is `swiftlint` issues. (Saves credits)

Reply:
The external image doesn't required here actually. SwiftLint is available within usual [macOS-12](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#linters) system.

So it could be just ran by `swiftlint --whatever-attribute`.